doc_admin/AdminManual.adoc

```diff
@@ -470,21 +583,24 @@ The following table shows the default value and the allowed ones:
 |SINGLEUSER |yes |yes, no
 |===
 
-OpenHospital, since its first version, allow a multi-user usage. It means that is possible to define different users, arrange them by group and set different rights in application usage; for instance, a user “receptionist” may be able to register a new patient but not to prescribe a therapy.
+OpenHospital, since its first version, allows for multi-user usage. It means that is possible to define different users, arrange them by group and set different rights in application usage; for instance, a user “receptionist” may be able to register a new patient but not to prescribe a therapy.
 
-OpenHospital has a predefined user called “admin” that have all rights to access all functionality in the application. When SINGLEUSER is set to YES (default) the program start with this user and all the functionalities are available.
+OpenHospital has a predefined user called “admin” that has all rights to access all the functionality of the application.
+When SINGLEUSER is set to YES (default) the program starts with this user and all the program functionality is available.
 
-If SINGLEUSER is set to NO the following login window will appear when start the program:
+If SINGLEUSER is set to NO the following login window appears when the program is started:
 
-image:extracted-media/media/image24.PNG[Login.PNG,width=305,height=148]
+image:image24.PNG[Login.PNG,width=305,height=148]
 
-The default password for “admin” user is “admin”. By typing his password in the white field and by pressing *[.underline]##S##ubmit*, you will see the main *_Menu_* with all functionalities enabled.
+The default password for “admin” user is “admin”. After typing this password in the white field and pressing *[.underline]##S##ubmit*, you will see the main *_Menu_* with all functionalities enabled.
 
-Once you’re logged in as “admin” user you can define users and group (see link:#_Users_and_Groups[Users and Groups (Users)]), or just stay with only that as a simple form of data protection.
+Once logged in as the “admin” user you can define users and groups (see the Users & Groups chapter in the User's Guide), or just continue with a simple form of data protection.
```

Comment:
this could have been replaced with an external link to the actual online documetation, like this?

[Users & Groups chapter in the User's Guide](https://github.com/informatici/openhospital-doc/blob/develop/doc_user/UserManual.adoc#15-users-groups-file-users)

Reply:
@mwithi Done; thanks for the catch.

Reply:
Thank you too @dbmalkovsky !